File: hw/calle/u3/hw28PrinciplesAplication/ComputerStoreJAVA/Products/src/main/java/ec/edu/espe/model/Product.java
package ec.edu.espe.model;

public class Product {
    private String name;
    private double basePrice;
    private double totalPrice;

    public Product(String name, double basePrice) {
        this.name = name;
        this.basePrice = basePrice;
    }

    public String getName() {
        return name;
    }
    public double getBasePrice() {
        return basePrice; 
    }
    public double getTotalPrice() {
        return totalPrice; 
    }
    public void setTotalPrice(double totalPrice) { 
        this.totalPrice = totalPrice;
    }
}

File: hw/calle/u3/hw28PrinciplesAplication/ComputerStoreJS/model/product.js
class Product {
    #name;
    #basePrice;
    #totalPrice;

    constructor(name, basePrice) {
        this.#name = name;
        this.#basePrice = basePrice;
        this.#totalPrice = 0;
    }

    getName() { return this.#name; }
    getBasePrice() { return this.#basePrice; }
    getTotalPrice() { return this.#totalPrice; }

    setTotalPrice(value) { this.#totalPrice = value; }
}

module.exports = Product;